db/migrate/20160422094212_add_index_unique_community_id_and_facebook_id.rb

```diff
@@ -0,0 +1,8 @@
+class AddIndexUniqueCommunityIdAndFacebookId < ActiveRecord::Migration
+  def change
+    # Remove the old facebook_id from index because we don't
+    # do searches with facebook id only
```

Comment:
Good point. We should now change the `sessions_controller` query to not use `community_memberships` but instead use the `community_id`. But, it's not changed yet so I'll keep the index there for a while.